documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
+Custom Resource Definitions (CRDs) extend the Kubernetes API, providing definitions to create and add custom resources to an {ProductPlatformName} cluster. Custom resources are created as instances of CRDs.
+
+For {ProductName}, CRDs introduce custom resources specific to {ProductName} to an {ProductPlatformName} cluster, such as Kafka, Kafka Connect, Kafka Mirror Maker, and users and topics.  CRDs provide configuration instructions, defining the schemas used to instantiate and manage the resources specific to {ProductName}. CRDs also allow {ProductName} resources to benefit from native {ProductPlatformName} features like CLI accessibility and configuration validation.
+
+CRDs require a one-time installation to a cluster. Depending on the cluster setup, installation typically requires cluster admin privileges.
+
+NOTE: `ClusterRoles` and `ClusterRoleBindings` to manage the CRDs are applied on installation to provide access for the Cluster Operator.
```

Comment:
Honestly, I'm still not convinced by this. I'm not sure anyone cares about it in this section. I think it would be much more interesting to point out that regular users will not have access to work with these resources and point to the chapter [2.9](https://strimzi.io/docs/master/full.html#assembly-getting-started-strimzi-admin-str) how to give it to them.

Reply:
Changed: Access to manage custom resources is limited to xref:assembly-getting-started-strimzi-admin-{context}[{ProductName} administrators].